lib/node_modules/@stdlib/math/base/special/lcmf/benchmark/benchmark.js

```diff
@@ -0,0 +1,54 @@
+/**
```

Comment:
@Harsh-Mathur-1503 The indentation of this file is off. It must use tabs instead of spaces.